src/jack.py

```diff
@@ -56,10 +56,29 @@
 NETSTARTING = _lib.JackTransportNetStarting
 """Waiting for sync ready on the network."""
 
+PROPERTY_CREATED = _lib.PropertyCreated
+"""A property was created.  See `Client.set_property_change_callback()`."""
+PROPERTY_CHANGED = _lib.PropertyChanged
+"""A property was changed.  See `Client.set_property_change_callback()`."""
+PROPERTY_DELETED = _lib.PropertyDeleted
+"""A property was deleted.  See `Client.set_property_change_callback()`."""
+
 _SUCCESS = 0
 _FAILURE = 1
 
 
+def _decode(cdata):
+    return _ffi.string(cdata).decode()
+
+
+# Get metadata constants from library
+for name in dir(_lib):
+    if name.startswith('JACK_METADATA_'):
+        globals()[name[5:]] = _decode(getattr(_lib, name))
```

Comment:
Thanks, that's a great trick!

However, I think i would like to be a bit more defensive in case those constants are not defined in the library. I wanted to do that at a later point, but I guess I can already do it now: 5d9b56541aa9659ecb87ea14b108574b0beb55bb